### ISSUE TITLE
[Accessibility] Some dialogs can't be called via shortcut when drop-down is opened

### ISSUE THREAD

**Issue by abariska** [musescore/MuseScore#17192]
### Issue type

Other type of issue

### Bug description

According to accessibility rules, drop-down should be closed and shortcut action should be done after pressing shortcut over the drop-down.

The issue can be reproduced with Playback setup dialog, but could be another issued dialogs

### Steps to reproduce

Set Playback setup dialog to a shortcut with Cmnd/Ctrl

1. Open any score
2. Open any drop-down (e.g. Instrument settings in Inspector or Properties) (input fields shouldn't be focused)
3. Press shortcut

### Screenshots/Screen recordings

https://user-images.githubusercontent.com/101250347/230978663-458a3f04-8210-4d10-82a3-e6af3abb98a5.mp4

### MuseScore Version

MuseScore 4.0.2 

### Regression

No.

### Operating system

macOS 13
